src-docs/src/views/guidelines/writing_examples.js

```diff
@@ -0,0 +1,730 @@
+            iconType="importAction"
+            title={<h2>No queries to profile</h2>}
+            body={
+              <p>
+                Enter a query, click the Profile button, and see the results
+                here.
+              </p>
+            }
+          />
+          <EuiSpacer />
+        </GuideRuleExample>
+      </GuideRule>
+      <GuideRule heading="" description="">
+        <GuideRuleExample
+          panelDisplay="block"
+          type="do"
+          text="Introduce a feature."
+        >
+          <EuiEmptyPrompt
+            iconType="importAction"
+            title={<h2>Introducing our table view</h2>}
+            body={
+              <p>
+                Explore your data inside Discover with our improved sorting and
+                filtering tools.
+              </p>
+            }
+            actions={
+              <EuiButton color="primary" fill>
+                Go to Discover
+              </EuiButton>
+            }
+          />
+        </GuideRuleExample>
+        <GuideRuleExample
+          panelDisplay="block"
+          type="do"
+          text="If using an empty for an error, explain next steps."
+        >
+          <EuiEmptyPrompt
+            iconType="importAction"
+            title={<h2>We couldn&apos;t log you in</h2>}
+            body={
+              <p>
+                Check your credentials and try again. If you still can&apos;t
+                log in, contact your system administrator.
+              </p>
+            }
+          />
+          <EuiSpacer />
+        </GuideRuleExample>
+      </GuideRule>
+
+      <EuiSpacer />
+
+      <EuiLink href="#/display/empty-prompt">
+        More empty prompts examples
+      </EuiLink>
+
+      <EuiHorizontalRule />
+
+      <EuiSpacer size="m" />
+
+      <EuiText>
+        <h2 id="labels">Labels</h2>
+
+        <p>
+          Avoid long labels, but don&apos;t sacrifice clarity. If needed, put
+          additional information in help text and tooltips.
+        </p>
+      </EuiText>
+
+      <GuideRule heading=" " description="">
+        <GuideRuleExample
+          panelDisplay="block"
+          type="do"
+          text="Use labels that say what the component does."
+        >
+          <EuiFormRow>
+            <EuiCheckbox
+              onChange={() => {}}
+              id={htmlIdGenerator()()}
+              label="Combine values in other bucket"
+            />
+          </EuiFormRow>
+          <EuiFormRow label="Bucket label">
+            <EuiFieldText />
+          </EuiFormRow>
+        </GuideRuleExample>
+        <GuideRuleExample
+          panelDisplay="block"
+          type="dont"
+          text="Use generic labels."
+        >
+          <EuiFormRow>
+            <EuiCheckbox
+              onChange={() => {}}
+              id={htmlIdGenerator()()}
+              label="Combine values in other bucket"
+            />
+          </EuiFormRow>
+          <EuiFormRow label="Bucket label">
```

Comment:
```suggestion
        <GuideRuleExample
          panelDisplay="block"
          type="do"
          text="Use specific labels."
        >
          <EuiFormRow>
            <EuiCheckbox
              onChange={() => {}}
              id={htmlIdGenerator()()}
              label="Limit incoming traffic"
            />
          </EuiFormRow>
          <EuiFormRow 
                      label="Traffic source"
                      helpText={
              <span>Enter CIDR or IP address, for example 192.168.132.6/22.</span>
            }>
```

I hope I've done this right. I'd prefer to get rid of the checkbox for this entirely, but I wasn't sure If I could, so I just changed the wording.